[docs] [drivers] Move node.js and csharp driver pages to Drivers and ORMs. (#14689)
yugabyte/yugabyte-db: docs/content/preview/reference/drivers/nodejs/postgres-pg-reference.md

> The [PostgreSQL node-postgres driver](https://node-postgres.com/) is the official Node.js driver for PostgreSQL which can be used to connect with YugabyteDB YSQL API. Because YugabyteDB YSQL API has is fully compatible with PostgreSQL node-postgres (pg) driver, it allows Node.js programmers to connect to the YugabyteDB database to execute DMLs and DDLs using the node-postgres APIs.

Reviewer: ```suggestion
The [PostgreSQL node-postgres driver](https://node-postgres.com/) is the official Node.js driver for PostgreSQL. The YSQL API is fully compatible with the PostgreSQL node-postgres (pg) driver, so you can connect to the YugabyteDB database to execute DMLs and DDLs using the node-postgres APIs.
```

$ npm install --save async
```

### Connect to YugabyteDB database

> Node.js applications can connect to and query the YugabyteDB database using the `pg` module. The `pg.Client` is the client we can create for the application  by passing the connection string as parameter in the constructor. The `Client.connect()` can be used to connect with database with that connection string. Use `new Client(<connection-string>)` to create a new client with the connection string.

Reviewer: This describes a bunch of code that isn't actually on this page - it's all on the drivers and orms page

Author: That is how the reference pages are structured from the beginning. Maybe during a new iteration we can work on modifying it better.
